tomcat8-nojsvc@.service

```diff
@@ -0,0 +1,36 @@
+# Systemd unit file for tomcat instances.
+#
+# To create clones of this service:
+# 0. systemctl enable tomcat8-nojsvc@name.service
+# 1. create catalina.base directory structure in
+#    /var/lib/tomcats/name
+# 2. set catalina base inside /etc/sysconfig/tomcat8@<name>
+
+[Unit]
+Description=Apache Tomcat 8 Web Application Container (No Jsvc)
+After=syslog.target network.target
+
+[Service]
+Type=forking
+
+Environment="NAME=%I"
+
+Environment=CATALINA_HOME=/opt/tomcat8
+Environment=JAVA_HOME=/usr/lib/jvm/jre-1.8.0
+Environment='CATALINA_OPTS=-Xms512M -Xmx1024M -server -XX:+UseParallelGC -Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom'
+Environment=CATALINA_PID=/opt/tomcat8/temp/tomcat-nojsvc-%I.pid
+Environment=TOMCAT_USER=tomcat
+
+EnvironmentFile=-/etc/sysconfig/tomcat8@%I
+
+PIDFile=/opt/tomcat8/temp/tomcat-nojsvc-%I.pid
```

Comment:
The PID file path can be set to be something like `/run/tomcat8-%i`.

I don't know what FHS thinks about this... You could use the `RuntimeDirectory` that is created/cleaned up automatically when the unit starts/stops.

For example, have a look at https://github.com/nethesis/dartagnan/blob/167993f666040135363943a4c22c6aeb80cb8503/porthos/root/etc/systemd/system/redis%40.service#L15

Reply:
New version of unit template no more use `PIDFile`